backend/src/Notifo.Domain.Integrations/Discord/DiscordServiceExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// ==========================================================================
+//  Notifo.io
+// ==========================================================================
+//  Copyright (c) Sebastian Stehle
+//  All rights reserved. Licensed under the MIT license.
+// ==========================================================================
+
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Notifo.Domain.Integrations.Discord;
```

Comment:
Check the namespace of other ServiceExtensions classes, I usually put them into "Microsoft.Extensions.DependencyInjection". it is a best practice to make it more discoverable and to void dozens and hundreds of usings in the Startup file.